src/Symfony/Bundle/FrameworkBundle/Console/Descriptor/JsonDescriptor.php

```diff
@@ -209,6 +209,7 @@ protected function getRouteData(Route $route): array
             'hostRegex' => '' !== $route->getHost() ? $route->compile()->getHostRegex() : '',
             'scheme' => $route->getSchemes() ? implode('|', $route->getSchemes()) : 'ANY',
             'method' => $route->getMethods() ? implode('|', $route->getMethods()) : 'ANY',
+            'priority' => $route->getPriority() ? $route->getPriority() : '',
```

Comment:
why sometimes you compare `0 !== $route->getPriority()` and sometimes only `$route->getPriority()`? 
maybe using the same pattern is better